react/Header/UserAccountMenu/UserAccountMenu.js

```diff
@@ -1,88 +1,215 @@
 import styles from './UserAccountMenu.less';
 
 import React, { PropTypes } from 'react';
+import classnames from 'classnames';
 
+import SearchIcon from '../../SearchIcon/SearchIcon';
 import ProfileIcon from '../../ProfileIcon/ProfileIcon';
 import HeartIcon from '../../HeartIcon/HeartIcon';
 import StarIcon from '../../StarIcon/StarIcon';
+import ThumbsUpIcon from '../../ThumbsUpIcon/ThumbsUpIcon';
+import employerLinkForLocale from '../employerLinkForLocale';
+import appendReturnUrl from '../appendReturnUrl';
+import { AUTHENTICATED, UNAUTHENTICATED, AUTH_PENDING } from '../authStatusTypes';
 
 const clearLocalStorage = () => {
   if (window && window.localStorage) {
     localStorage.clear();
   }
 };
 
-export default function UserAccountMenu({ linkRenderer }) {
+export default function UserAccountMenu({ locale, authenticationStatus, linkRenderer, returnUrl, activeTab }) {
   return (
     <ul className={styles.root}>
-      <li>
+      <li className={classnames(styles.smallDeviceOnly, activeTab === 'Job Search' && styles.activeTab)}>
+        {
+          linkRenderer({
+            className: styles.item,
+            href: '/',
+            children: [
+              <span key="label">Job Search</span>,
+              <SearchIcon
+                key="icon"
+                className={classnames(styles.icon, styles.jobSearch)}
+                svgClassName={styles.iconSvg}
+              />
+            ]
+          })
+        }
+      </li>
+      <li className={classnames(activeTab === 'Profile' && styles.activeTab)}>
         {
           linkRenderer({
             className: styles.item,
             href: '/profile/',
             children: [
-              <span key="label">Your profile</span>,
-              <ProfileIcon key="icon" className={styles.icon} svgClassName={styles.iconSvg} />
+              <span key="label">Profile</span>,
+              <ProfileIcon
+                key="icon"
+                className={classnames(styles.icon, styles.profile)}
+                svgClassName={styles.iconSvg}
+              />
             ]
           })
         }
       </li>
-      <li>
+      <li className={classnames(styles.aboveSmallDevice, activeTab === 'Saved Searches' && styles.activeTab)}>
```

Comment:
IMO it's overkill if we don't have a static typing system, given there's very little harm in getting it wrong.